setup.py

```diff
@@ -39,6 +39,7 @@
     "bcrypt==3.1.7",
     "certifi>=2020.12.5",
     "ciso8601==2.1.3",
+    "h11==0.12.0",
```

Comment:
This is a subdep constrain right?

If so, it should go into the `package_constraints.txt` file (please note, the `CONSTRAINT_BASE` constant in `script/gen_requirements_all.py` needs to be adjusted for that).

Does it have to be pinned that hard? Or is an `h11>=0.12.0` sufficient as well?

Reply:
Anything of 0.12 or later should work. I will update according to your comments within a couple of hours

Reply:
Done